x-pack/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -148,11 +148,21 @@ export interface ConfusionMatrix {
   other_predicted_class_doc_count: number;
 }
 
+export interface AucRocCurveItem {
+  fpr: number;
+  threshold: number;
+  tpr: number;
+}
+
 export interface ClassificationEvaluateResponse {
   classification: {
-    multiclass_confusion_matrix: {
+    multiclass_confusion_matrix?: {
       confusion_matrix: ConfusionMatrix[];
     };
+    auc_roc?: {
+      curve?: AucRocCurveItem[];
+      value: number;
```

Comment:
I think it is worth displaying the actual values for each class somewhere on the page.

Reply:
Fixed in 6bcca25. The AUC values are not part of the legend labels: 

![image](https://user-images.githubusercontent.com/230104/107359708-4c8fc180-6ad5-11eb-9fd1-afe554f1034b.png)